logstash-core-event-java/lib/logstash-core-event-java/version.rb

```diff
@@ -5,4 +5,4 @@
 # Note to authors: this should not include dashes because 'gem' barfs if
 # you include a dash in the version string.
 
-LOGSTASH_CORE_EVENT_JAVA_VERSION = "3.0.0.dev"
+LOGSTASH_CORE_EVENT_JAVA_VERSION = "5.0.0-alpha1.snapshot1"
```

Comment:
the packaging name and the version name are different I remember now why there is a gsub below.